Source/ACE/Managers/GuidManager.cs

```diff
@@ -14,25 +14,33 @@ public static class GuidManager
         // players
         private const uint playerMin = 0x50000001;
         private const uint playerMax = 0x59999999;
-        private static uint player = 0x50000001;
+        private static uint player   = 0x50000001;
+
+        // admins
+        private const uint adminMin = 0x80000001;
+        private const uint adminMax = 0x89999999;
```

Comment:
Copied from player, changed from 5 to 8 based on what I'd seen in used in the past.